self-api/src/main/java/com/selfxdsd/api/User.java

```diff
@@ -27,8 +27,8 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
- * @todo #278:30min Implement and test Organizations for Gitlab
- *  provider.
+ * @todo #278:30min Continue with GitlabOrganizations by implementing and
```

Comment:
@criske YOu have to also change the parent ticket number, otherwise the todo will not close, I think :D